src/lib/glocal/detect-exec-layout.spec.ts

```diff
@@ -105,21 +118,21 @@ describe('local available detection', () => {
       runningLocalTool: false,
     })
   })
-  it('if just node_module/dir missing, discounts being available', () => {
-    ctx.fs.remove('node_modules/a')
-    expect(ctx.detectExecLayout()).toMatchObject({
-      nodeProject: true,
-      toolProject: true,
-      toolCurrentlyPresentInNodeModules: false,
-      runningLocalTool: false,
-    })
-  })
+  //  it('if just node_module/dir missing, discounts being available', () => {
+  //    ctx.fs.remove('node_modules/a')
+  //    expect(ctx.detectExecLayout()).toMatchObject({
+  //      nodeProject: true,
+  //     toolProject: true,
+  //      toolCurrentlyPresentInNodeModules: false,
+  //      runningLocalTool: false,
+  //    })
+  //  })
 })
 
 describe('running locally detection', () => {
   beforeEach(installTool)
   it('if process script path matches path to tool in project bin then considered running locally', () => {
-    expect(ctx.detectExecLayout({ scriptPath: ctx.fs.path('node_modules/.bin/a') })).toMatchObject({
```

Comment:
Shouldn't we maintain a posix test that sets the path like so? It would show that we're using realPathSync etc.

But IIUC such a test needs to be behind a if-posix condition.

Reply:
(Ok, just did this one myself now)